src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private Object[] array;
+    private int arraySize;
+
+    public ArrayList() {
+        array = new Object[DEFAULT_SIZE];
+        arraySize = 0;
+    }
+
+    private Object[] grow() {
```

Comment:
let’s move private methods to the bottom of the file, or at least after the place where we call them

Reply:
fixed